### PR TITLE
Standalone Order by, Skip/Offset, and Limit

### PR DESCRIPTION
**Pull request by JoelBergstrand**
Adds documentation for standalone Order by, Skip/Offset, and Limit clause and introduction of the Offset keyword. Implementation of [CIP-168](https://docs.google.com/document/d/1fKnZHtJcP7A9ulbVJV3JfitM1AcMs4kz_hdAxDoNc2U/edit?usp=sharing)

Questions:

- Should the ORDER BY, LIMIT, SKIP, OFFSET pages be collected into one page, or an additional page added, to reflect that they now can be used as one clause now?
- Should the Clauses index page be modified to reflect that ORDER BY, LIMIT, SKIP, OFFSET can be used as a standalone clause instead of only as subclauses?

Corresponding [PR](https://github.com/neo-technology/neo4j/pull/26710)

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1027.surge.sh

New pages:
- [OFFSET](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses/offset)

Updated pages:
- [Clauses](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses)
- [LIMIT](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses/limit)
- [ORDER BY](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses/order-by)
- [SKIP](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses/skip)
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1027.surge.sh

New pages:
- [OFFSET](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses/offset)

Updated pages:
- [Supported mandatory GQL features](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/appendix/gql-conformance/supported-mandatory)
- [Clauses](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses)
- [LIMIT](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses/limit)
- [ORDER BY](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses/order-by)
- [SKIP](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses/skip)
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

File: modules/ROOT/pages/clauses/limit.adoc [hunk 1]
```diff
@@ -145,3 +145,29 @@ Writes `locked` property on one node and return that node:
...
+`LIMIT` can be used as a standalone clause, or in conjunction with `ORDER BY` or `SKIP`/`OFFSET`.
+
+.Query
+[source, cypher, indent=0]
+----
+MATCH (n)
+ORDER BY n.name DESC
+SKIP 2
+LIMIT 2
+RETURN collect(n.name) AS names
```
Comment (gem-neo4j) [inline modules/ROOT/pages/clauses/limit.adoc:194]: This query isn't really using it standalone though right? Would standalone not look more like: MATCH (n) LIMIT 2 RETURN collect(n.name) AS names?

**Review by gem-neo4j — COMMENTED** [review]
Great! I think we can discuss with Jens a little more on your questions :)

File: modules/ROOT/pages/clauses/limit.adoc (hunk 1 above)
Comment (JoelBergstrand) [reply modules/ROOT/pages/clauses/limit.adoc:194]: I think there could be a better way to word it, or possibly have to examples. One with only limit "standalone" and one example of them together as in the vast majority of cases it doesn't make sense to use it by itself without at least and order by.

Comment (JoelBergstrand) [reply modules/ROOT/pages/clauses/limit.adoc:194]: So, one could argue that a separate page or section would be more useful to show the existence of `Order by n.name SKIP 1 LIMIT 1` as a standalone clause

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1027.surge.sh

Updated pages:
- [Supported mandatory GQL features](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/appendix/gql-conformance/supported-mandatory)
- [Clauses](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses)
- [LIMIT](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses/limit)
- [ORDER BY](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses/order-by)
- [SKIP](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses/skip)
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

File: modules/ROOT/pages/clauses/limit.adoc [hunk 2]
```diff
@@ -141,7 +151,50 @@ Writes `locked` property on one node and return that node:
...
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="1*<m"]
+|===
+| names
+| ["Andy", "Bernard"]
+|Rows: 1
+|===
```
Comment (JoelBergstrand) [inline modules/ROOT/pages/clauses/limit.adoc:182]: @JPryce-Aklundh Here I think we should clarify that the ordering is not guaranteed, so users do not believe they will get ascending ordering by default.

Comment (JoelBergstrand) [reply modules/ROOT/pages/clauses/limit.adoc:182]: We seem to have this disclaimer for SKIP which I believe is applicable here as well. 

_Neo4j does not guarantee the results generated by SKIP/OFFSET. The only clause that guarantees a specific row order is [ORDER BY](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses/order-by/)._

File: modules/ROOT/pages/clauses/skip.adoc [hunk 3]
```diff
@@ -70,42 +78,110 @@ SKIP 1
...
+| names
+| ["David", "Charlotte"]
+|Rows: 1
+|===
+
+[role=label--new-5.24]
+[[offset-synonym]]
+== `OFFSET` as a synonym to `SKIP`
+
+`OFFSET` was introduced as part of Cypher's xref:appendix/gql-conformance/index.adoc[] and can be used as a synonym to `SKIP`.
```
Comment (JoelBergstrand) [inline modules/ROOT/pages/clauses/skip.adoc:169]: This is much better!

**Comment by JoelBergstrand** [comment]
Great cleanup with the new images and examples!

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1027.surge.sh

Updated pages:
- [Supported mandatory GQL features](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/appendix/gql-conformance/supported-mandatory)
- [Clauses](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses)
- [LIMIT](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses/limit)
- [ORDER BY](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses/order-by)
- [SKIP](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses/skip)
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Review by gem-neo4j — APPROVED** [review]
This looks good to me :D

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1027.surge.sh

Updated pages:
- [Supported mandatory GQL features](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/appendix/gql-conformance/supported-mandatory)
- [Clauses](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses)
- [LIMIT](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses/limit)
- [ORDER BY](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses/order-by)
- [SKIP](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/clauses/skip)
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1027.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**